apps/mobile/src/app/(auth)/sign-up.tsx

```diff
@@ -0,0 +1,13 @@
+import { Text } from 'react-native';
+
+import { ScreenWrapper } from '@app/components/components';
+
+const SignUpScreen = () => {
+	return (
+		<ScreenWrapper>
+			<Text>Sign up</Text>
```

Comment:
We should have sign up screen. The data should be saved in DB after sign up
Use react-hook-form

Reply:
I thought this will be a task for the students